pkg/operator/controller/ingress/status.go

```diff
@@ -50,7 +50,7 @@ type expectedCondition struct {
 
 // syncIngressControllerStatus computes the current status of ic and
 // updates status upon any changes since last sync.
-func (r *reconciler) syncIngressControllerStatus(ic *operatorv1.IngressController, deployment *appsv1.Deployment, deploymentRef metav1.OwnerReference, pods []corev1.Pod, service *corev1.Service, operandEvents []corev1.Event, wildcardRecord *iov1.DNSRecord, dnsConfig *configv1.DNS, platformStatus *configv1.PlatformStatus) (error, bool) {
+func (r *reconciler) syncIngressControllerStatus(ic *operatorv1.IngressController, deployment *appsv1.Deployment, deploymentRef metav1.OwnerReference, pods []corev1.Pod, currentLBService *corev1.Service, operandEvents []corev1.Event, wildcardRecord *iov1.DNSRecord, dnsConfig *configv1.DNS, platformStatus *configv1.PlatformStatus) (error, bool) {
```

Comment:
this is a nit pick, but should we really change the existing variable name `service` to `currentLBService`? In context, all of the other variable names are `deployment`, `pod`, `ic`, not `currentDeployment` or `currentIC`. Seems a little inconsistent.

I think the going assumption is that all parameters are "current" objects and not desired. I think it's better to stay consistent with these variable names to avoid confusion.

Reply:
@gcs278  done